macros/GroupData.json

```diff
@@ -1258,6 +1258,93 @@
+                            "PerformanceNavigationTiming.toJSON()",
+                            "PerformanceObserver.observe()",
+                            "PerformanceObserver.disconnect()",
+                            "PerformanceResourceTiming.toJSON()" ],
+            "properties": [ "Performance.navigation",
+                            "Performance.timing",
+                            "PerformanceEntry.name",
+                            "PerformanceEntry.entryType",
+                            "PerformanceEntry.startTime",
+                            "PerformanceEntry.duration",
+                            "PerformanceLongTaskTiming.attribution",
+                            "PerformanceNavigationTiming.domComplete",
+                            "PerformanceNavigationTiming.domContentLoadedEventEnd",
+                            "PerformanceNavigationTiming.domContentLoadedStart",
+                            "PerformanceNavigationTiming.domInteractive",
+                            "PerformanceNavigationTiming.loadEventEnd",
+                            "PerformanceNavigationTiming.loadEventStart",
+                            "PerformanceNavigationTiming.redirectCount",
+                            "PerformanceNavigationTiming.type",
+                            "PerformanceNavigationTiming.unloadEventEnd",
+                            "PerformanceNavigationTiming.unloadEventStart",
+                            "PerformanceResourceTiming.connectEnd",
+                            "PerformanceResourceTiming.connectStart",
+                            "PerformanceResourceTiming.decodedBodySize",
+                            "PerformanceResourceTiming.domainLookupEnd",
+                            "PerformanceResourceTiming.domainLookupStart",
+                            "PerformanceResourceTiming.encodedBodySize",
+                            "PerformanceResourceTiming.fetchStart",
+                            "PerformanceResourceTiming.initiatorType",
+                            "PerformanceResourceTiming.nextHopProtocol",
+                            "PerformanceResourceTiming.redirectEnd",
+                            "PerformanceResourceTiming.redirectStart",
+                            "PerformanceResourceTiming.requestStart",
+                            "PerformanceResourceTiming.responseEnd",
+                            "PerformanceResourceTiming.responseStart",
+                            "PerformanceResourceTiming.secureConnectStart",
+                            "PerformanceResourceTiming.transferSize",
+                            "PerformanceResourceTiming.workerStart",
+                            "TaskAttributionTiming.containerId",
+                            "TaskAttributionTiming.containerName",
+                            "TaskAttributionTiming.containerSrc",
+                            "TaskAttributionTiming.containerType" ],
+            "events":     [ "Performance.onresourcetimingbufferfull" ]
```

Comment:
This needs to be the name of the event "resourcetimingbufferfull" to create a link to the event page:
https://developer.mozilla.org/en-US/docs/Web/Events/resourcetimingbufferfull